docker/compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  tuono:
+    container_name: tuono-source-container
+    hostname: tuono-source-container
+    build:
+      context: ../
+      dockerfile: ./docker/Dockerfile
+      args:
+        RUST_VERSION: 1.83
+        BASE_OS: bookworm
+        NODE_VERSION: 22.11.0
+    image: tuono-source-image
+    ports:
+      - "3000:3000" # frontend 
+      - "3001:3001" # backend
```

Comment:
I think we can remove this port binding. The port `3001` should never be accessed by the client.

> Note: port `3001` is the server exposed by vite. The client interact with it by calling `localhost:3000/vite-server/*`. Our dev server then just proxies all the requests to vite.